repairnator/docker-images/bears-dockerimage/logback.xml

```diff
@@ -14,15 +14,15 @@
 	</appender>
 
 	<logger name="fr.inria.spirals.jtravis.helpers" level="info" />
-	<logger name="fr.inria.spirals.repairnator" level="info" />
+	<logger name="fr.inria.spirals.repairnator" level="${LOG_LEVEL:-info}" />
```

Comment:
I don't understand this notation. The ":-info" is a default value?

Reply:
Yes.